src/js/StyledIcon.js

```diff
@@ -4,15 +4,17 @@ import styled, { css } from 'styled-components';
 import { colorStyle } from 'grommet-styles';
 
 import { defaultProps } from './default-props';
-import { parseMetricToNum } from './utils';
+import { parseMetricToNum, useIconPad } from './utils';
```

Comment:
The "use" prefix makes it sound like a custom hook. But, this isn't a React component here. I think we should just use "iconPad" as a name.

Reply:
That makes sense, I'll adjust.